test/system/src/main/java/io/pravega/test/system/framework/DockerBasedTestExecutor.java

```diff
@@ -44,8 +44,9 @@
     public static final int DOCKER_CLIENT_PORT = 2375;
     private final static String IMAGE = "java:8";
     private final AtomicReference<String> id = new AtomicReference();
-    private final DockerClient client = DefaultDockerClient.builder().uri("http://" +
-            getConfig("masterIP", "Invalid Master IP") + ":" + DOCKER_CLIENT_PORT).build();
+    private final String masterIp = Utils.isAwsExecution() ? getConfig("awsMasterIP", "Invalid Master IP").trim() : getConfig("masterIP", "Invalid Master IP");
```

Comment:
Is it possible to use the property "masterIP" instead of "awsMasterIP" for AWS based system test run?

Reply:
No, the system property `masterIP` is set to invalid in the config phase, but we only get the aws public ip of master , in the exec phase, hence using another variable.  Using the same `masterIP` did not work. Hence, create another variable.